### PR TITLE
Updates to be more similar to getenvoy

### PR DESCRIPTION
**Pull request by codefromthecrypt**
```bash
$ goreleaser-v0.166.0  build --snapshot --single-target --rm-dist
--snip--
      • building                  binary=/Users/adrian/oss/getmesh/dist/getmesh_darwin_amd64/getmesh
   • build succeeded after 10.36s
```

### CODE REVIEW

File: .goreleaser.yml [hunk 1]
```diff
@@ -1,21 +1,16 @@
...
       - amd64
-archives:
-  - format: tar.gz
-    name_template: '{{ .Binary }}_{{ .Os }}_{{ .Arch }}{{ if .Arm }}v{{ .Arm }}{{ end }}{{ if .Mips }}_{{ .Mips }}{{ end }}'
-    files: [""] # necessary to exclude any files except the binary
-checksum:
-  name_template: 'checksums.txt'
-snapshot:
-  name_template: "{{ .Tag }}-next"
+      - arm64
```
Comment (codefromthecrypt) [inline .goreleaser.yml:13]: added this as it is no effort. Ack the downloads for istio won't likely exist, so I can remove it.

**Review by codefromthecrypt — COMMENTED** [review]
I will fix the find/replace fuzz on 1.17.6

**Comment by mathetake** [comment]
I can fix tests for you since it's going to be tough 😄

**Comment by mathetake** [comment]
maybe better revert 1.7.6 change

**Comment by mathetake** [comment]
in anyway I would do that in https://github.com/tetratelabs/getmesh/pull/43

**Comment by codefromthecrypt** [comment]
ok @mathetake I removed the istio version skew things

**Comment by mathetake** [comment]
OK for e2e failure.. will fix later!

**Pull request merged**